### PR TITLE
Wdt 644 -- output tool log file location at startup

### PR DESCRIPTION
**Pull request by jwgish**

### CODE REVIEW

File: core/src/main/python/prepare_model.py [hunk 1]
```diff
@@ -84,7 +85,7 @@ def main():
         _outputdir = model_context.get_output_dir()
         model_files = model_context.get_model_file()
 
-        obj = ModelPreparer(model_files, model_context, _outputdir)
+        obj = ModelPre https://github.com/jwgish/weblogic-deploy-tooling/pull/new/WDT-644parer(model_files, model_context, _outputdir)
```
Comment (ddsharpe) [inline core/src/main/python/prepare_model.py:88]: typo?

File: core/src/main/python/wlsdeploy/exception/exception_helper.py [hunk 2]
```diff
@@ -195,7 +196,7 @@ def create_validate_exception(key, *args, **kwargs):
     """
     arg_list, error = _return_exception_params(*args, **kwargs)
     arg_len = len(arg_list)
-    if error is not None:
+    if error  https://github.com/jwgish/weblogic-deploy-tooling/pull/new/WDT-644is not None:
```
Comment (ddsharpe) [inline core/src/main/python/wlsdeploy/exception/exception_helper.py:199]: typo?

File: core/src/main/python/wlsdeploy/exception/exception_helper.py [hunk 3]
```diff
@@ -295,7 +296,7 @@ def create_yaml_exception(key, *args, **kwargs):
     elif len(arg_list) > 0:
         ex = JYamlException(key, arg_list)
     else:
-        ex = JYamlException(key)
+        ex =  https://github.com/jwgish/weblogic-deploy-tooling/pull/new/WDT-644JYamlException(key)
```
Comment (ddsharpe) [inline core/src/main/python/wlsdeploy/exception/exception_helper.py:299]: typo?

**Review by ddsharpe — CHANGES_REQUESTED** [review]

File: core/src/main/python/prepare_model.py (hunk 1 above)
Comment (jwgish) [reply core/src/main/python/prepare_model.py:88]: I don't know how the heck this got in here, and I can't figure out how to get rid of it.

**Comment by jwgish** [comment]
Abandoning -- can't figure out how to kill the typos that creeped in through some mysterious mechanism.

**Pull request closed without merging**